src/devices/essentials.ts

```diff
@@ -0,0 +1,138 @@
+import fz from '../converters/fromZigbee';
+import * as exposes from '../lib/exposes';
+import {logger} from '../lib/logger';
+import * as tuya from '../lib/tuya';
+import {Definition, Fz, Tz} from '../lib/types';
+
+const e = exposes.presets;
+const ea = exposes.access;
+
+const NS = 'zhc:essentials';
+
+const essentialsValueConverter = {
+    battery: {
+        from: (value: number, meta: Fz.Meta) => {
+            return value > 130 ? 100 : value < 70 ? 0 : ((value - 70) * 1.7).toFixed(1);
+        },
+    },
+    faultCode: {
+        from: (value: string, meta: Fz.Meta) => {
+            logger.warning(`zigbee-herdsman-converters:essentialsThermostat: ERROR CODE received: ${JSON.stringify(value)}`, NS);
+            return value;
+        },
+    },
+    away_setting: {
+        from: (value: number[]) => {
+            return {
+                year: value[0] + 2000,
+                month: value[1],
+                day: value[2],
+                hour: value[3],
+                minute: value[4],
+                away_preset_temperature: Number((value[5] / 2).toFixed(1)),
+                away_preset_days: (value[6] << 8) | value[7],
+            };
+        },
+        to: (
+            value: {
+                year: number;
+                month: number;
+                day: number;
+                hour: number;
+                minute: number;
+                away_preset_temperature: number;
+                away_preset_days: number;
+            },
+            meta: Tz.Meta,
+        ) => {
+            const output = new Uint8Array(8);
+            // byte 0 - Start Year (0x00 = 2000)
+            // byte 1 - Start Month
+            // byte 2 - Start Day
+            // byte 3 - Start Hour
+            // byte 4 - Start Minute
+            // byte 5 - Temperature (1~59 = 0.5~29.5°C (0.5 step))
+            // byte 6-7 - Duration in Hours (0~2400 (100 days))
+            output[0] = value.year > 2000 ? value.year - 2000 : value.year; // year
+            output[1] = value.month; // month
+            output[2] = value.day; // day
+            output[3] = value.hour; // hour
+            output[4] = value.minute; // min
+            output[5] = Math.round(value.away_preset_temperature * 2);
+            output[7] = value.away_preset_days & 0xff;
+            output[6] = value.away_preset_days >> 8;
+            logger.debug(JSON.stringify({'send to tuya': output, 'value was': value}), NS);
+        },
+    },
+};
+
+const essentialsThermostat: Definition = {
+    fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_i48qyn9s'}],
+    model: 'Essentials Zigbee Radiator Thermostat',
```

Comment:
Is there a better model number written on the device/product page?

Reply:
"essentials Heizkörperthermostat Zigbee Smart Home"

It's barely distinguishable from other non-Zigbee thermostats, and I haven't seen any significant model number or something like that on the packaging.

Reply:
Then I would use `_TZE200_i48qyn9s` as the model.

Reply:
Done